viewer/reveal.api.md

```diff
@@ -779,8 +779,20 @@ export interface Image360 {
 // @public
 export interface Image360Collection {
     readonly image360Entities: Image360[];
+    off(event: 'image360Entered', callback: Image360EnteredDelegate): void;
+    // (undocumented)
+    off(event: 'image360Exited', callback: Image360ExitedDelegate): void;
+    on(event: 'image360Entered', callback: Image360EnteredDelegate): void;
+    // (undocumented)
+    on(event: 'image360Exited', callback: Image360ExitedDelegate): void;
 }
 
+// @public
+export type Image360EnteredDelegate = () => void;
```

Comment:
Makes sense that the entered delegate also includes the Image360 that is entered, right?

Reply:
would the API users have any usage with Image360, I am just understanding what usecase would it be helpful for 🤔